code/modules/surgery/organs/organ.dm

```diff
@@ -268,29 +268,34 @@ var/list/organ_cache = list()
 /obj/item/organ/emp_act(severity)
 	if(!(status & ORGAN_ROBOT))
 		return
-	switch(severity)
-		if(1.0)
-			take_damage(0,20)
-			return
-		if(2.0)
-			take_damage(0,7)
-			return
-		if(3.0)
```

Comment:
Oh really? ... Should I take this as a project to update across the board @Fox-McCloud ?